src/model/neuroevolution/individual.py

```diff
@@ -97,8 +95,8 @@ def random_neuron_weights_mutation(individual: "ChildIndividual") -> None:
         random_layer = individual.get_random_layer()
         random_neuron_index = np.random.randint(0, random_layer.weights.shape[0])
         # TODO decide the range of new random weights
-        random_layer.weights[random_neuron_index] = np.random.random(
-            (random_layer.weights.shape[1])
+        random_layer.weights[random_neuron_index] = (
+            np.random.random((random_layer.weights.shape[1])) * 6 - 3
```

Comment:
It would be more clear to use `np.random.uniform` with bounds.

Reply:
Sure, done